pytorch_lightning/tuner/lr_finder.py

```diff
@@ -200,6 +192,14 @@ def lr_find(
     if trainer.progress_bar_callback:
         trainer.progress_bar_callback.enable()
 
+    # Update lr attr if required
+    if update_attr:
+        lr = lr_finder.suggestion()
+
+        # TODO: log lr.results to self.logger
+        lightning_setattr(model, lr_attr_name, lr)
+        log.info(f'Learning rate set to {lr}')
```

Comment:
rank zero info?

Reply:
I guess it is not running inside any process, so not required?